k8s/namespaces/dg/dg-docassembly/prod.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: dg-docassembly
+spec:
+  values:
+    java:
+      environment:
+        DOCMOSIS_TEMPLATES_ENDPOINT: "https://docmosis.platform.hmcts.net"
+        DOCMOSIS_CONVERT_ENDPOINT: "https://docmosis.platform.hmcts.net/rs/convert"
+        DOCMOSIS_RENDER_ENDPOINT: "https://docmosis.platform.hmcts.net/rs/render"
+        ENABLE_FORM_DEFINITION_ENDPOINT: "false"
+        ENABLE_TEMPLATE_RENDITION_ENDPOINT: "true"
```

Comment:
We don't need this line as this is same as in the https://github.com/hmcts/dg-docassembly-api/blob/f4751578feadc6da0936c7e06001234e4a2bd58b/charts/dg-docassembly/values.yaml#L44